src/HealthChecks.RavenDB/RavenDBHealthCheck.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Raven.Client.Documents;
+using Raven.Client.ServerWide.Operations;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HealthChecks.RavenDB
+{
+    /// <summary>
+    /// Health check for RavenDB.
+    /// </summary>
+    /// <seealso cref="Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheck" />
+    public class RavenDBHealthCheck : IHealthCheck
+    {
+        private readonly string _connectionString;
+        private readonly string _specifiedDatabase;
+
+        /// <summary>
```

Comment:
We don't have comments on HealthChecks because probably is not used directly never.

Reply:
Ok